CognitiveSearch.Skills/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Please remove this file from your checkin, as well as other binary files.

Reply:
removed